Makefile

```diff
@@ -112,6 +112,18 @@ pause-container-release: pause-container
 	mkdir -p "$(shell pwd)/out"
 	@docker save ${PAUSE_CONTAINER_IMAGE}:${PAUSE_CONTAINER_TAG} > "$(shell pwd)/out/${PAUSE_CONTAINER_TARBALL}"
 
+# Variable to determine branch/tag of amazon-ecs-cni-plugins
+revision=master
+cni-sources:
+	@git clone https://github.com/aws/amazon-ecs-cni-plugins.git --branch $(revision)
+
+cni-plugins: cni-sources
```

Comment:
The decoupling for development is agreeable since we are bundling the cni binaries within the agent anyway. 
We could go with this approach but this definitely makes building the plugins a two phase approach.